doc/classes/RenderingDevice.xml

```diff
@@ -2044,28 +2044,34 @@
 		<constant name="DYNAMIC_STATE_STENCIL_REFERENCE" value="64" enum="PipelineDynamicStateFlags" is_bitfield="true">
 		</constant>
 		<constant name="INITIAL_ACTION_CLEAR" value="0" enum="InitialAction">
+			Start rendering and clear the whole framebuffer, region or not (supply appropriate parameters when using this action).
 		</constant>
 		<constant name="INITIAL_ACTION_CLEAR_REGION" value="1" enum="InitialAction">
+			Start rendering and clear the framebuffer in the specified region (supply appropriate parameters when using this action).
 		</constant>
 		<constant name="INITIAL_ACTION_CLEAR_REGION_CONTINUE" value="2" enum="InitialAction">
+			Continue rendering and clear the framebuffer in the specified region (supply appropriate parameters when using this action).
 		</constant>
 		<constant name="INITIAL_ACTION_KEEP" value="3" enum="InitialAction">
+			Start rendering, but keep attached color texture contents.
 		</constant>
 		<constant name="INITIAL_ACTION_DROP" value="4" enum="InitialAction">
+			Start rendering, ignore what is there; write above it.
 		</constant>
 		<constant name="INITIAL_ACTION_CONTINUE" value="5" enum="InitialAction">
+			Continue rendering. Framebuffer must have been left in [constant FINAL_ACTION_CONTINUE] state as the final action previously.
 		</constant>
 		<constant name="INITIAL_ACTION_MAX" value="6" enum="InitialAction">
 			Represents the size of the [enum InitialAction] enum.
 		</constant>
 		<constant name="FINAL_ACTION_READ" value="0" enum="FinalAction">
-			Store the texture for reading and make it read-only if it has the [constant TEXTURE_USAGE_SAMPLING_BIT] bit (only applies to color, depth and stencil attachments).
+			Store the texture for reading and make it read-only if it has the [constant TEXTURE_USAGE_SAMPLING_BIT] bit (only applies to color, depth and stencil attachments). This allows attached textures to be read again, but depth buffer contents will be dropped (they can't be read from).
```

Comment:
I'm not sure if the remark about the depth buffer is true. While we read from copies of the depth buffer if applicable, and do so before we reach the end of the render pipeline and not after, the depth buffer can be handed to external processes.

A valid use case here is XR where the depth buffer can be read from as part of reprojection. So as far as I know we don't discard it if the final action is set to read.

@clayjohn do you have any idea if this is correct?

Reply:
```suggestion
			Store the texture for reading and make it read-only if it has the [constant TEXTURE_USAGE_SAMPLING_BIT] bit (only applies to color, depth and stencil attachments).
```
This remark isn't true. FinalAction's are separated for depth and color. If the depth's FinalAction is set to ``FINAL_ACTION_READ`` it will be prepared for reading and will not be dropped.